circle.sh

```diff
@@ -2,23 +2,17 @@
 
 case "$1" in
   pre_machine)
-    # ensure correct level of parallelism
```

Comment:
This check doesn't seem to be valuable anymore. It'll be simpler to remove it since my java11 build is parallelism 1.